File: src/main/kotlin/neo4j/FilteredInfo.java
package neo4j;

import java.util.List;

public class FilteredInfo {
    public static List<String> FILTERED_NODES = List.of(
        "FunctionScope",
        "LLVMIRLanguage",
        "LocalScope",
        "TranslationUnitDeclaration",
        "OverlayEdge",
        "UnknownLanguage",
        "UnknownType",
        "Scope",
        "Type"
    );

    public static List<String> FILTERED_EDGES = List.of(
        "ASSIGNED_TYPES",
        "CANDIDATES",
        "LANGUAGE",
        "SCOPE", // this is useful (maybe) but i cant stand it popping up in neo4j right now
        "TYPE",
        "TYPE_OBSERVERS",
        "SECONDARY_TYPES"
    );
}

File: src/main/kotlin/lving/backend/cpg/neo4j/FilteredInfo.kt
package lving.backend.cpg.neo4j

val FILTERED_NODES = listOf(
    "FunctionScope",
    "LLVMIRLanguage",
    "LocalScope",
    "TranslationUnitDeclaration",
    "OverlayEdge",
    "UnknownLanguage",
    "UnknownType",
    "Scope",
    "Type"
)

val FILTERED_EDGES = listOf(
    "ASSIGNED_TYPES",
    "CANDIDATES",
    "LANGUAGE",
    "SCOPE",
    "TYPE",
    "TYPE_OBSERVERS",
    "SECONDARY_TYPES"
)